src/components/VExpansionPanel/VExpansionPanelContent.ts

```diff
@@ -3,19 +3,40 @@ import { VExpandTransition } from '../transitions'
 import Bootable from '../../mixins/bootable'
 import Toggleable from '../../mixins/toggleable'
 import Rippleable from '../../mixins/rippleable'
-import { inject as RegistrableInject } from '../../mixins/registrable'
+import { Registrable, inject as RegistrableInject } from '../../mixins/registrable'
 
 import VIcon from '../VIcon'
+import VExpansionPanel from '.'
+
+import mixins, { ExtractVue } from '../../util/mixins'
+import Vue, { VNode } from '../../../node_modules/vue'
```

Comment:
Another one

Reply:
dammit. seems like it's the vscode 1.25 update that's causing it